app/src/main/java/com/example/sharewhatyoucanproject/rdashboard/rdetailedPosts/RdetailsViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.sharewhatyoucanproject.rdashboard.rdetailedPosts
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.example.sharewhatyoucanproject.models.PostModel
+import com.google.firebase.firestore.FirebaseFirestore
+
+class RDetailsViewModel(
```

Comment:
Could you make this name more descriptive?

Reply:
done